spec/support/fetchable_behavior.rb

```diff
@@ -81,78 +86,83 @@
         expect(described_class.create!(valid_attributes.merge(url: url)).url).
-          to eq("http://www.nps.gov/UsaGovLovesToCapitalize")
+          to eq('http://agency.gov/UsaGovLovesToCapitalize')
       end
     end
 
-    context "when URL is missing trailing slash for a scheme+host URL" do
-      let(:url) { "http://www.nps.gov" }
-      it "should append a /" do
+    context 'when URL is missing trailing slash for a scheme+host URL' do
+      let(:url) { 'http://agency.gov' }
+
+      it 'appends a /' do
         expect(described_class.create!(valid_attributes.merge(url: url)).url).
-          to eq("http://www.nps.gov/")
+          to eq('http://agency.gov/')
       end
     end
 
-    context "when URL contains duplicate leading slashes in request" do
-      let(:url) { "http://www.nps.gov//hey/I/am/usagov/and/love/extra////slashes.shtml" }
-      it "should collapse the slashes" do
+    context 'when URL contains duplicate leading slashes in request' do
+      let(:url) { 'http://agency.gov//hey/I/am/usagov/and/love/extra////slashes.shtml' }
+
+      it 'collapses the slashes' do
         expect(described_class.create!(valid_attributes.merge(url: url)).url).
-          to eq("http://www.nps.gov/hey/I/am/usagov/and/love/extra/slashes.shtml")
+          to eq('http://agency.gov/hey/I/am/usagov/and/love/extra/slashes.shtml')
       end
     end
 
     context "when URL doesn't have a protocol" do
-      let(:url) { "www.nps.gov/sdfsdf" }
+      let(:url) { 'www.nps.gov/sdfsdf' }
 
-      it "prepends it with https://" do
-        expect{ record.valid? }.to change{ record.url }.from(url).
-          to('https://www.nps.gov/sdfsdf')
+      it 'prepends it with https://' do
+        expect { record.valid? }.to change(record, :url).
+          from(url).to('https://www.nps.gov/sdfsdf')
       end
     end
 
     context 'when the url contains query parameters' do
       let(:url) { 'http://www.irs.gov/foo?bar=baz' }
 
       it 'retains the query parameters' do
-        expect{ record.valid? }.not_to change{ record.url }
+        expect { record.valid? }.not_to change(record, :url)
       end
     end
 
     context 'when the url requires escaping' do
-      let(:url) { "https://www.foo.gov/my_url’s_weird!" }
+      let(:url) { 'https://www.foo.gov/my_url’s_weird!' }
 
       it 'escapes the url' do
-        expect{ record.valid? }.
-          to change{ record.url }.from(url).to("https://www.foo.gov/my_url%E2%80%99s_weird!")
+        expect { record.valid? }.
+          to change(record, :url).
+          from(url).to('https://www.foo.gov/my_url%E2%80%99s_weird!')
       end
 
       context 'when the url is already escaped' do
-        let(:url) { "https://www.foo.gov/my_url%E2%80%99s_weird!" }
+        let(:url) { 'https://www.foo.gov/my_url%E2%80%99s_weird!' }
 
         it 'does not re-escape the url' do
-          expect{ record.valid? }.not_to change{ record.url }
+          expect{ record.valid? }.not_to(change{ record.url })
```

Comment:
The `RSpec/ExpectChange` cop is complaining about this, but this is actually the style we should enforce for consistency with the rest of our specs. So we should update the Rubocop configuration accordingly:
```
RSpec/ExpectChange
  Enabled: true
  EnforcedStyle: block
```
You'll need to revert the related spec changes you made.

Also, the parentheses should be omitted, because `not_to` falls under the rule to "Always omit parentheses for methods that are part of an internal DSL (e.g., Rake, Rails, RSpec)" (https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens-internal-dsl).

Reply:
Changed and reverted.